client/components/Filter/FilterSettings.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { VscSettings } from 'react-icons/vsc'
```

Comment:
@AStashevskaya Also looks like `react-icons/vsc` is badly bundled and `93.5%` of package is unused:

<img width="969" alt="image" src="https://user-images.githubusercontent.com/12561302/114562261-a09e5a00-9c6e-11eb-9415-60fbab1764bd.png">

Could we use just free `svg` icons from https://www.iconfinder.com